messages/2.1.1.txt

```diff
@@ -0,0 +1,7 @@
+=============
+Release 2.1.1
+=============
+
+PackageSync now can choose to synchronize user setting or not.

```

Comment:
Typos. Set to `You now have support to sync your PackageSync settings across devices as well. Do use this with caution though!!!`